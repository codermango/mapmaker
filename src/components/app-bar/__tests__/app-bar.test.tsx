import React from "react";
import renderer from "react-test-renderer";
import AppBar from "../app-bar";

test("renders correctly", () => {
  const tree = renderer.create(<AppBar />);
  expect(tree).toMatchSnapshot();
});
