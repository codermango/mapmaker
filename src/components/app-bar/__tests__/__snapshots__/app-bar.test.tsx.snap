// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="appBar"
>
  In order for the tool to work properly, please make sure your broswer zoom is 100%
</div>
`;
